source/install/cloud-changelog.md

```diff
@@ -4,12 +4,58 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-06-29](#release-2023-06-29)
 - [Release 2023-06-13](#release-2023-06-13)
 - [Release 2023-05-31](#release-2023-05-31)
 - [Release 2023-05-01](#release-2023-05-01)
 - [Release 2023-04-21](#release-2023-04-21)
 - [Release 2023-03-29](#release-2023-03-29)
-- [Release 2023-03-20](#release-2023-03-20)
+
+## Release 2023-06-29
+
+### Improvements
+
+#### User Interface (UI)
+ - Added support to specify different desktop notification sounds per channel.
+ - Calls: Ringing sounds can be enabled/disabled and selected in the Desktop Notifications preferences panel.
+ - Bumped the pre-packaged version of NPS to 1.3.2.
+ - Apps Bar is now enabled by default for on-prem servers. ``ExperimentalSettings.EnableAppBar`` was also renamed to ``ExperimentalSettings.DisableAppBar``. See more details at:
+    - https://docs.mattermost.com/welcome/what-changed-in-v70.html#apps-bar-beta
+    - https://forum.mattermost.com/t/channel-header-plugin-changes/13551
+
+#### Administration
+ - Boards has been moved from a product back to a plugin and is now disabled by default. ``EnablePublicSharedBoards`` Boards product setting was removed from the config and the **System Console**.
+ - Playbooks is again shipped as a prepackaged plugin, with version 1.36.1.
+ - Removed setting the Channel Export and Apps plugins enabled by default.
+ - Added a new ``ConfigurationWillBeSaved`` plugin hook which is invoked before the configuration object is committed to the backing store.
+ - Insights has been disabled for all new instances and for existing servers that upgrade to v8.0. The Insights feature flag can be enabled with environment variables via `MM_FEATUREFLAGS_INSIGHTSENABLED=true`.
+ - Admins can now specify index names to ignore while purging indexes from Elasticsearch with the ``ElasticsearchSettings.IgnoredPurgeIndexes`` setting.
+ - Added an option to use the German HPNS notification proxy.
+ - New flags were added to the database migrate command as following:
+
+    - ``auto-recover``: If the migration plan receives an error during migrations, this command will try to rollback migrations already applied within the plan. This option is not recommended to be added without reviewing migration plan. You can review the plan by combining ``--save-plan`` and ``--dry-run`` flags.
+    - ``save-plan``: The plan for the migration will be saved into the file store so that it can be used for reviewing the plan or to be used for downgrading
```

Comment:
```suggestion
    - ``save-plan``: The plan for the migration will be saved into the file store so that it can be used for reviewing the plan or to be used for downgrading.
```